src/styles/style.css

```diff
@@ -33,14 +44,12 @@ header::after {
 header::after {
   animation: arrow-anim 1s infinite alternate ease-out;
   font-weight: bold;
-  left: 50%;
-  position: absolute;
+  content: '￬';
+  z-index: 2;
+  position: relative;
```

Comment:
In header::after try changing the **position: absolute** and add **left: 49%** it would remove normal flow.